pkg/cri/config/config.go

```diff
@@ -88,6 +88,14 @@ type ContainerdConfig struct {
 	DiscardUnpackedLayers bool `toml:"discard_unpacked_layers" json:"discardUnpackedLayers"`
 }
 
+// LogConfig contains toml config related to log
+type LogConfig struct {
+	// LogLimit is the limit number of the log lines that allowed to be written per second
+	LogLimit int `toml:"log_limit" json:"logLimit"`
```

Comment:
in the config this will be under "log" structure right? so we drop `log_` prefix in `log_limit` and switch `logLimit` to just `limit`